lib/rubocop/cop/style/multiline_hash_brace_layout.rb

```diff
@@ -4,39 +4,45 @@
 module RuboCop
   module Cop
     module Style
-      # This cop checks that the closing brace in an a hash literal is
-      # symmetrical with respect to the opening brace and the hash
-      # elements.
+      # This cop checks that the closing brace in a hash literal is either
+      # on the same line as the last hash element, or a new line.
+      #
+      # When using the `symmetrical` (default) style:
       #
       # If a hash's opening brace is on the same line as the first element
       # of the hash, then the closing brace should be on the same line as
       # the last element of the hash.
       #
-      # If a hash's opening brace is on a separate line from the first
-      # element of the hash, then the closing brace should be on the line
+      # If a hash's opening brace is on the line above the first element
+      # of the hash, then the closing brace should be on the line below
+      # the last element of the hash.
+      #
+      # When using the `new_line` style:
+      #
+      # The closing brace of a multi-line hash literal must be on the line
       # after the last element of the hash.
       #
       # @example
       #
-      #     # bad
-      #     { a: 'a',
-      #       b: 'b'
-      #     }
-      #
-      #     # bad
-      #     {
-      #       a: 'a',
-      #       b: 'b' }
-      #
-      #     # good
-      #     { a: 'a',
-      #       b: 'b' }
-      #
-      #     #good
-      #     {
-      #       a: 'a',
-      #       b: 'b'
-      #     }
+      #     # bad with symmetrical, good with new_line
+      #     [ a: 1,
```

Comment:
I'll fix the syntax in this comment. These should be hashes not arrays. The line-breaks are correct though. The `new_line` style only enforces that the closing brace is on a new line.